uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -421,13 +451,13 @@ define([
        * @return {Promise} with true value if period found else rejected false
        */
       function checkAndSetAbsencePeriod(date) {
-        var formattedDate = moment(date).format(vm.uiOptions.userDateFormat.toUpperCase());
+        var formattedDate = moment(date).format(this.uiOptions.userDateFormat.toUpperCase());
```

Comment:
`var formattedDate = moment(date).format(sharedSettings.serverDateFormat);`
Not related to this ticket, but please change as we decided to keep the dateFormat in the sharedSettings constant only, and remove rest.